src/controllers/productController.js

```diff
@@ -22,9 +22,48 @@ const create = async (req, res) => {
   res.redirect("/api/v1/products");
 };
 
-const showNewForm = (req, res) => {
-  res.render("products/showNewForm");
+const edit = async (req, res) => {
```

Comment:
Please rename this function to show the intent what it does like `editProduct` or `updateProduct`

Reply:
done